src/sdk/map_api_types.js

```diff
@@ -493,8 +493,20 @@ function get_all_chunks_blocks(chunks) {
         (_.flatMapDeep(chunks, chunk => chunk.frags.map(frag => frag.blocks)))
     );
 }
+/**
+ * 
+ * @param {nb.Chunk[]} chunks 
+ * @returns {nb.Part[]}
+ */
+function get_all_chunk_parts(chunks) {
+    return /** @type {nb.Part[]} */ (
+        /** @type {unknown} */
+        (_.flatMapDeep(chunks, chunk => chunk.parts))
+    );
```

Comment:
Since this is just one level then we don't need the deep flat map variant and can use the native array method, which shouldn't have all that hassle with types (I think so):
```suggestion
    return chunks.flatMap(chunk => chunk.parts);
```